src/state-mutation.js

```diff
@@ -59,6 +60,17 @@ export const makeUpdateReverseRelationship = (
       )
     ) {
       return foreignResources;
+    } else if (Array.isArray(foreignResourceRel) && !newRelation) {
+      const relIdx = foreignResourceRel.findIndex(item => (
+        item.id === resource.id
+      ));
+
+      if (foreignResourceRel[relIdx]) {
+        const deletePath = [idx, 'relationships', singular, 'data', relIdx];
+        return imm(foreignResources).del(deletePath).value();
```

Comment:
@egeriis I made immutable deletion. One thing I need to call `value()` for the modified state.